Duplicati/Library/Backend/OpenStack/OpenStackStorage.cs

```diff
@@ -72,6 +72,7 @@ public class OpenStackStorage : IBackend, IStreamingBackend
             new KeyValuePair<string, string>("Infomaniak Swiss Backup cluster 2", "https://swiss-backup02.infomaniak.com/identity/v3"),
             new KeyValuePair<string, string>("Infomaniak Swiss Backup cluster 3", "https://swiss-backup03.infomaniak.com/identity/v3"),
             new KeyValuePair<string, string>("Infomaniak Public Cloud 1", "https://api.pub1.infomaniak.cloud/identity/v3"),
+            new KeyValuePair<string, string>("Catalyst Cloud (NZ)", "https://api.nz-por-1.catalystcloud.io:5000//v3"),

```

Comment:
Does this really need two //'s? If that's correct, and you can post something showing it's working I will push for this to be merged.

Reply:
Good spotting. I have fixed that, and added another of our regions.